### PR TITLE
Fix hardcoded dashboard names in the code

### PR DESCRIPTION
**Pull request by asmorodskyi**
Fix hardcoded dashboard names in the code

### CODE REVIEW

File: openqabot/types/aggregate.py [hunk 1]
```diff
@@ -171,7 +171,7 @@ def __call__(
                 continue
 
             full_post["openqa"]["__DASHBOARD_INCIDENTS_URL"] = ",".join(
-                f"https://dashboard.qam.suse.de/incident/{inc}"
+                f"{QEM_DASHBOARD}incident/{inc}"
```
Comment (mimi1vx) [inline openqabot/types/aggregate.py:174]: missing `/` and this also in incidents.py and some tests

**Review by mimi1vx — CHANGES_REQUESTED** [review]

File: tests/test_openqabot_simple.py [hunk 2]
```diff
@@ -94,7 +94,7 @@ def test_passed(mock_runtime, mock_openqa_passed, caplog):
     )
     bot = OpenQABot(args)
 
-    responses.add(responses.PUT, "http://dashboard.qam.suse.de/bar", json={"id": 234})
+    responses.add(responses.PUT, f"{QEM_DASHBOARD}/bar", json={"id": 234})
```
Comment (mimi1vx) [inline tests/test_openqabot_simple.py:97]: there is one more '/' than needed

File: tests/test_smeltsync.py [hunk 3]
```diff
@@ -82,7 +83,7 @@ def reply_callback(request):
 
     responses.add_callback(
         responses.PATCH,
-        re.compile(r"http://dashboard.qam.suse.de/api/incidents"),
+        re.compile(f"{QEM_DASHBOARD}/api/incidents"),
```
Comment (mimi1vx) [inline tests/test_smeltsync.py:86]: also here is one more `/` than needed

File: tests/test_incsyncres.py [hunk 4]
```diff
@@ -1,10 +1,11 @@
 from collections import namedtuple
 import logging
+from urllib.parse import urlparse
 import pytest
 import responses
-from urllib.parse import urlparse
 import openqabot.incsyncres
 from openqabot.incsyncres import IncResultsSync
+from openqabot.approver import QEM_DASHBOARD
```
Comment (asmorodskyi) [inline tests/test_incsyncres.py:8]: I am not sure about this one but I haven't found a way to import variable directly from __init__ :thinking:

File: tests/test_openqabot_simple.py (hunk 2 above)
Comment (asmorodskyi) [reply tests/test_openqabot_simple.py]: you right , fixed

File: tests/test_smeltsync.py (hunk 3 above)
Comment (asmorodskyi) [reply tests/test_smeltsync.py]: fixed

File: tests/test_incsyncres.py (hunk 4 above)
Comment (mimi1vx) [reply tests/test_incsyncres.py:8]: `from openqabot import QEM_DASHBOARD` ....

Comment (asmorodskyi) [reply tests/test_incsyncres.py]: done

File: tests/test_openqabot_simple.py [hunk 5]
```diff
@@ -5,7 +5,7 @@
 import pytest
 import responses
 
-from openqabot.openqabot import OpenQABot
+from openqabot.openqabot import OpenQABot, QEM_DASHBOARD
```
Comment (mimi1vx) [inline tests/test_openqabot_simple.py:8]:
~~~
from openqabot.openqabot import OpenQABot
from openqabot.openqabot import QEM_DASHBOARD
~~~

Comment (kalikiana) [reply tests/test_openqabot_simple.py:8]: Why do we need this split? Is this something we can unify?

Comment (asmorodskyi) [reply tests/test_openqabot_simple.py]: done

**Review by kalikiana — APPROVED** [review]

**Pull request merged**